docs/source/deploying-volttron/secure-deployment-considerations.rst

```diff
@@ -92,3 +92,23 @@ This approach has some limitations, including:
 - There could be cases where tampering adjusts values to incorrect but in-bounds values
   which would not be detected.
 
+
+Monitoring RabbitMQ Server
+==========================
+
+RabbitMQ server may be configured to run as a systemd service and allow systemd to monitor the
+status of the service. It can be further configured to detect and restart the RabbitMQ service
+if it crashes. VOLTTRON agents have the ability to detect when the RabbitMQ server crashes/disconnects
+and reconnect when it comes back up again. In this deployment setup, VOLTTRON platform will not
+start/stop the RabbitMQ server instead allow systemd service to start/stop RabbitMQ server.
+
+VOLTTRON also has mechanism to monitor RabbitMQ server periodically and restart if a crash is detected
```

Comment:
Is this the same setup as the one above where we don't start/stop the server?  Are the two combined?  Perhaps in the monitoring section it would be good to start with the different options and then go to the different paragraphs?

Reply:
Fixed